domain/services/controller.go

```diff
@@ -42,9 +41,10 @@ import (
 
 // ControllerServices provides access to the services required by the apiserver.
 type ControllerServices struct {
```

Comment:
```suggestion
type ControllerFactory struct {
```
This way we'd be consistent with `ModelFactory` and `ProviderFactory`.

Reply:
I'll be changing the _others_ in a future patch.

Reply:
It's the inverse, the naming should be services, not factories. I didn't fix all locations because I'm already tackling enough naming refactors.